feat: add pages to guide through the reuse declaration procedure (#807)
italia/developers.italia.it: it/riuso/dichiarazione.md

    url: /it/riuso/pubblicazione
---

<!-- markdownlint-disable no-inline-html -->

Secondo quanto previsto dalle [Linee guida su acquisizione e riuso di software per la PA](https://docs.italia.it/AgID/linee-guida-riuso-software/lg-acquisizione-e-riuso-software-per-pa-docs/it/bozza/attachments/allegato-d-guida-alla-modifica-di-software-open-source-preso-a-riuso-o-di-terzi.html)
e dal [Piano Triennale per l'Informatica nella Pubblica
Amministrazione](https://docs.italia.it/italia/piano-triennale-ict/pianotriennale-ict-doc/it/2020-2022/capitolo-1-servizi/cosa-devono-fare-le-pa.html),
non appena una Pubblica Amministrazione italiana acquisisce un software di
titolarità di un altro Ente **è necessario effettuare una dichiarazione per
comunicare all’Ente cedente che il software è stato preso in riuso** ed è
attualmente utilizzato.

Questa segnalazione permette all’Ente cedente di avere contezza di chi stia
attivamente utilizzando un dato software e **facilitarne in questo modo
l'interazione.**

<div class="show-if-not-repo">
  Le operazioni necessarie per effettuare la dichiarazione
  variano in base all’esistenza o meno del software all’interno del catalogo.

  <h3>

>     Se il software non è presente nel catalogo di Developers Italia

Reviewer: ```suggestion
    Se il software è presente nel catalogo di Developers Italia
```

  </h3>
</div>

<div class="show-if-repo">
  <p>
    Le operazioni necessarie per effettuare la dichiarazione sono:
  </p>
</div>

1. Visitare il <strong><a id="repo" rel="noreferrer noopener nofollow external" target="_blank" href="">repository relativo al software in questione</a></strong>
2. All’interno del repository è possibile **comunicare con i manutentori del
   software** utilizzando uno degli strumenti di interazione messi a disposizione
   dalla piattaforma. Tra gli approcci possibili, questi due sono i più utilizzati:

   * richiesta di inserire il _nome_ e il _codiceIPA_ del proprio ente
     all’interno del file `publiccode.yml` attraverso lo strumento di ticketing (_issue_)
   * inserimento del _nome_ e _codiceIPA_ del proprio ente direttamente
     all’interno del campo `usedBy` del `publiccode.yml` tramite una _Pull Request_.

In questo modo il manutentore del software potrà concludere le operazioni in
autonomia e il nome del proprio Ente verrà visualizzato all’interno del catalogo.

<div class="show-if-not-repo">
  <h3>

>     Se il software è presente nel catalogo di Developers Italia.

Reviewer: ```suggestion
    Se il software non è presente nel catalogo di Developers Italia
```

    url: /it/riuso/pubblicazione
---

<!-- markdownlint-disable no-inline-html -->

Secondo quanto previsto dalle [Linee guida su acquisizione e riuso di software per la PA](https://docs.italia.it/AgID/linee-guida-riuso-software/lg-acquisizione-e-riuso-software-per-pa-docs/it/bozza/attachments/allegato-d-guida-alla-modifica-di-software-open-source-preso-a-riuso-o-di-terzi.html)
e dal [Piano Triennale per l'Informatica nella Pubblica
Amministrazione](https://docs.italia.it/italia/piano-triennale-ict/pianotriennale-ict-doc/it/2020-2022/capitolo-1-servizi/cosa-devono-fare-le-pa.html),
non appena una Pubblica Amministrazione italiana acquisisce un software di
titolarità di un altro Ente **è necessario effettuare una dichiarazione per
comunicare all’Ente cedente che il software è stato preso in riuso** ed è
attualmente utilizzato.

Questa segnalazione permette all’Ente cedente di avere contezza di chi stia
attivamente utilizzando un dato software e **facilitarne in questo modo
l'interazione.**

<div class="show-if-not-repo">
  Le operazioni necessarie per effettuare la dichiarazione
  variano in base all’esistenza o meno del software all’interno del catalogo.

>   <h3>

Reviewer: ```suggestion
  <h2>
```

    Se il software è presente nel catalogo di Developers Italia

>   </h3>

Reviewer: ```suggestion
  </h2>
```

Reviewer: Can we try with h2? I don't particularly like the h3 style applied in the page

            messi a disposizione dalla piattaforma.
          </p>
        </div>
      </div>
    </div>
  </div>

  <div class="col-12">
    <div class="card-wrapper card-space">
      <div class="card card-bg">
        <div class="card-body">
          <div class="categoryicon-top">
            <span class="circled-step">3</span>
          </div>
          <p class="card-title">
            Richiedere l'inserimento di <em>nome</em> e <em>codiceIPA</em> del proprio Ente
            nel file <code class="language-plaintext highlighter-rouge">publiccode.yml</code>
            attraverso lo strumento di ticketing  (<em>issue</em>) oppure
            aggiungerli direttamente all’interno del campo
            <code class="language-plaintext highlighter-rouge">usedBy</code>
            del <code class="language-plaintext highlighter-rouge">publiccode.yml</code>
            tramite una <em>Pull Request</em>.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>

In questo modo il manutentore del software potrà concludere le operazioni in
autonomia e il nome del proprio Ente verrà visualizzato all’interno del catalogo.

<div class="show-if-not-repo">

>   <h3>

Reviewer: Same as above

    Se il software non è presente nel catalogo di Developers Italia
  </h3>

  <div class="callout note">
    <div class="callout-title"><svg class="icon"><use xlink:href="/assets/svg/sprite.svg#it-info-circle"></use></svg>Note a riguardo</div>
    <p>
      Si ricorda che la nuova modalità di riuso definita nelle
      <a href="{{ site.url_lineeguidariuso }}">Linee Guida</a>
      prevede che il processo di <a href="/it/riuso/acquisizione">acquisizione</a>
      avvenga attraverso il <a href="/it/software">catalogo del software di Developers Italia</a>.
    </p>
  </div>

  <p>
    Nel caso in cui il software preso a riuso non dovesse ancora essere
    entrato all’interno del catalogo per questioni tecniche oppure dovesse essere
    in procinto di terminare le operazioni di <a href="/it/riuso/pubblicazione">pubblicazione</a>
    <strong>è comunque possibile portare a termine questa dichiarazione.</strong>
  </p>
</div>

**Tramite gli strumenti di comunicazione** messi a disposizione da Developers Italia,

> quali ad esempio Slack oppure il Forum è sempre possibile avviare una

Reviewer: ```suggestion
quali ad esempio Slack oppure il Forum, è sempre possibile avviare una
```

> discussione e il **Team di Developers Italia interverrà per aiutare a completare

Reviewer: ```suggestion
discussione e il **[Team di Developers Italia](/it/contatti) interverrà per aiutare a completare
```

---

> title: Dichiarazione di presa a riuso di software

Reviewer: ```suggestion
title: Dichiarazione di riuso di software
```

lang: it
ref:
  en:
layout: internal-long
subtitle:
see_also:
  - label: Acquisizione di software
    url: /it/riuso/acquisizione
  - label: Pubblicazione di software
    url: /it/riuso/pubblicazione
---

<!-- markdownlint-disable no-inline-html -->

> Secondo quanto previsto dalle [Linee guida su acquisizione e riuso di software per la PA](https://docs.italia.it/AgID/linee-guida-riuso-software/lg-acquisizione-e-riuso-software-per-pa-docs/it/bozza/attachments/allegato-d-guida-alla-modifica-di-software-open-source-preso-a-riuso-o-di-terzi.html)

Reviewer: ```suggestion
Secondo quanto previsto dalle [Linee guida su acquisizione e riuso di software per la PA](https://docs.italia.it/italia/developers-italia/lg-acquisizione-e-riuso-software-per-pa-docs/it/stabile/attachments/allegato-d-guida-alla-modifica-di-software-open-source-preso-a-riuso-o-di-terzi.html)
```

e dal [Piano Triennale per l'Informatica nella Pubblica
Amministrazione](https://docs.italia.it/italia/piano-triennale-ict/pianotriennale-ict-doc/it/2020-2022/capitolo-1-servizi/cosa-devono-fare-le-pa.html),
non appena una Pubblica Amministrazione italiana acquisisce un software di
titolarità di un altro Ente **è necessario effettuare una dichiarazione per
comunicare all’Ente cedente che il software è stato preso in riuso** ed è
attualmente utilizzato.

Questa segnalazione permette all’Ente cedente di avere contezza di chi stia
attivamente utilizzando un dato software e **facilitarne in questo modo
l'interazione.**

<div class="show-if-not-repo">
  Le operazioni necessarie per effettuare la dichiarazione
  variano in base all’esistenza o meno del software all’interno del catalogo.

  <h2>

>     Se il software è presente nel catalogo di Developers Italia

Reviewer: ```suggestion
    Se il software è presente nel catalogo
```

  </h2>
</div>

<div class="show-if-repo">
  <p>
    Le operazioni necessarie per effettuare la dichiarazione sono:
  </p>
</div>

<div class="row">
  <div class="col-12 col-lg-6">
    <div class="card-wrapper card-space">
      <div class="card card-bg no-after">
        <div class="card-body d-flex align-items-center">
          <div class="categoryicon-top">
            <span class="circled-step">1</span>
          </div>
          <p>

>             Visitare il <strong><a id="repo" rel="noreferrer noopener nofollow external" target="_blank" href="">repository relativo al software in questione</a></strong>

Reviewer: ```suggestion
            Visitare il <strong><a id="repo" rel="noreferrer noopener nofollow external" target="_blank" href="">repository relativo al software in questione.</a></strong>
```

            Richiedere l'inserimento di <em>nome</em> e <em>codiceIPA</em> del proprio Ente
            nel file <code class="language-plaintext highlighter-rouge">publiccode.yml</code>
            attraverso lo strumento di ticketing  (<em>issue</em>) <strong>oppure</strong>
            aggiungerli direttamente all’interno del campo
            <code class="language-plaintext highlighter-rouge">usedBy</code>
            del <code class="language-plaintext highlighter-rouge">publiccode.yml</code>
            tramite una <em>Pull Request</em>.
          </p>
        </div>
      </div>
    </div>
  </div>

  <div class="col-12 col-lg-6">
    <div class="card-wrapper card-space">
      <div class="card card-bg no-after">
        <div class="card-body d-flex align-items-center">
          <div class="categoryicon-top">
            <span class="circled-step">4</span>
          </div>
          <p>
            Il manutentore del software potrà concludere le operazioni in
            autonomia e il nome del proprio Ente verrà visualizzato all’interno
            del catalogo.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>

<div class="show-if-not-repo">
  <h2>

>     Se il software non è presente nel catalogo di Developers Italia

Reviewer: ```suggestion
    Se il software non è presente nel catalogo
```
